src/mimeparser.rs

```diff
@@ -910,6 +921,11 @@ pub(crate) struct Report {
     original_message_id: String,
     /// Additional-Message-IDs
     additional_message_ids: Vec<String>,
+
+    /// MDNs should contain the same autodelete timer value as used in
+    /// the message they reference. It is used to determine autodelete
+    /// timer support.
```

Comment:
This sounds great, but I didn't see that implemented to show the user if the peer doesn't support autodelete.  Or did I miss this?  I completely understand if this is not for this PR but rather a followup though.

Reply:
If the other side does not support autodelete, it will return MDN without the timer, so the timer will be disabled. There is no explicit notification in the form of "user does not support timers". It will show up as "user has disabled the timer".